src/woo_search/search_index/tasks.py

```diff
@@ -14,6 +17,22 @@
 logger = logging.getLogger(__name__)
 
 
+def _download_document(download_url: str) -> None | str:
+    try:
+        response = requests.get(download_url)
+        response.raise_for_status()
+    except requests.RequestException as err:
+        logger.exception("Could not download the document.", err)
+        return
+
+    try:
+        return base64.b64encode(response.content).decode("ascii")
+    except binascii.Error as err:
+        logger.exception("Could not transform content to base64.", err)
+
+    return
```

Comment:
```suggestion
    assert False, "unreachable"
```
Would this be acceptable? I personally have no clue how we could test the response.content not being anything but binary data?